spec/controllers/settings_controller_spec.rb

```diff
@@ -1,28 +1,23 @@
 require 'spec_helper'
 
-describe SessionsController do
+describe SettingsController, type: :controller do
```

Comment:
Yep! We talked about that in class. I added them in on Friday -> https://github.com/benwoody/shrtnr/commit/2a9a3d664f5fe788c79cd20a5e9fe3638e1a1b1e